src/Database/Relations/DeferOneOrMany.php

```diff
@@ -48,13 +51,16 @@ public function withDeferred($sessionKey)
 
         // Unbind (Remove)
         $newQuery->whereNotExists(function($query) use ($sessionKey) {
+            if(DbDongle::getDriver() == 'sqlsrv') {
```

Comment:
Without this, the query will not execute on sql server. it needs the inital select *